float32 vx
float32 vy
float32 vz
string frame_id
bool frame_locked
---
bool success
string message
